ext/sysvsem/sysvsem.stub.php

```diff
@@ -6,9 +6,9 @@ final class SysvSemaphore
 {
 }
 
-function sem_get(int $key, int $max_acquire = 1, int $perm = 0666, bool $auto_release = true): SysvSemaphore|false {}
+function sem_get(int $key, int $max_acquire = 1, int $permissions = 0666, bool $auto_release = true): SysvSemaphore|false {}
 
-function sem_acquire(SysvSemaphore $semaphore, bool $nowait = false): bool {}
+function sem_acquire(SysvSemaphore $semaphore, bool $no_wait = false): bool {}
```

Comment:
Wait! This parameter seems very similar to `$blocking` in `msg_send()` (https://www.php.net/manual/en/function.msg-send.php). Would it be worth to call it `$no_blocking`?

Reply:
No strong opinion here. Looks like this flag corresponds to IPC_NOWAIT. If we go for `$no_blocking`, I'd use `$non_blocking`.

Reply:
> I'd use $non_blocking.

Ah, I felt that something is missing from `$no_blocking`, but I didn't realize what. 🤦 

Anyway, I went with `$non_blocking`, because I think people rarely have to check the underlying library/flag values, so it seems more useful for me to use a more standard terminology, especially if we have a precedent in `msg_send()`. :)